src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,51 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import lotto.LottoNumberCreationStrategy;
+
+public final class LottoNumbers {
+
+  public static final int LOTTO_NUMBER_COUNT = 6;
+
+  private final List<LottoNumber> lottoNumbers;
+
+  public LottoNumbers(LottoNumberCreationStrategy lottoNumberCreationStrategy) {
+    lottoNumbers = lottoNumberCreationStrategy.create();
+  }
+
+  public LottoNumbers(String[] winningNumbers) {
+    this.lottoNumbers = Arrays.stream(winningNumbers)
+        .map(Integer::parseInt)
+        .map(LottoNumber::new)
```

Comment:
```suggestion
        .map(LottoNumber::valueOf)
```
이렇게 표현할 수 있도록 만들면 어떨까요?

Reply:
빠르게 만들다보니 리팩토링할 포인트를 미처 발견하지 못했네요! 정적 팩토리 메서드 도입하겠습니다!